<ui version="4.0" >
 <class>PathPage</class>
 <widget class="QWidget" name="PathPage" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>437</width>
    <height>220</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>Geometry Directory:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" >
    <widget class="QLineEdit" name="m_geometryDir" >
     <property name="toolTip" >
      <string>Directory where all geometry files will be saved</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2" >
    <widget class="QToolButton" name="m_setGeometryDir" >
     <property name="text" >
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QLabel" name="label_4" >
     <property name="text" >
      <string>Textures Directory:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" >
    <widget class="QLineEdit" name="m_textureDir" >
     <property name="toolTip" >
      <string>Directory where all texture files will be saved</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2" >
    <widget class="QToolButton" name="m_setTextureDir" >
     <property name="text" >
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" >
    <widget class="QLabel" name="label_3" >
     <property name="text" >
      <string>Shader Directory:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" >
    <widget class="QLineEdit" name="m_shaderDir" >
     <property name="toolTip" >
      <string>Directory where all shader files will be saved</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2" >
    <widget class="QToolButton" name="m_setShaderDir" >
     <property name="text" >
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" >
    <widget class="QLabel" name="label_2" >
     <property name="text" >
      <string>Animation Directory:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" >
    <widget class="QLineEdit" name="m_animationDir" >
     <property name="toolTip" >
      <string>Directory where all animation files will be saved</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2" >
    <widget class="QToolButton" name="m_setAnimDir" >
     <property name="text" >
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" >
    <widget class="QLabel" name="label_5" >
     <property name="text" >
      <string>Materials Directory:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1" >
    <widget class="QLineEdit" name="m_materialDir" >
     <property name="toolTip" >
      <string>Directory where all material files will be saved</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2" >
    <widget class="QToolButton" name="m_setMaterialDir" >
     <property name="text" >
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" >
    <widget class="QLabel" name="label_7" >
     <property name="text" >
      <string>Scenegraph Directory:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1" >
    <widget class="QLineEdit" name="m_sceneGraphDir" >
     <property name="toolTip" >
      <string>Directory where all scenegraph files will be saved</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2" >
    <widget class="QToolButton" name="m_setSceneGraphDir" >
     <property name="text" >
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" >
    <widget class="QLabel" name="label_8" >
     <property name="text" >
      <string>Effects Directory:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1" >
    <widget class="QLineEdit" name="m_effectsDir" >
     <property name="toolTip" >
      <string>Directory where all effect files will be saved</string>
     </property>
    </widget>
   </item>
   <item row="6" column="2" >
    <widget class="QToolButton" name="m_setEffectsDir" >
     <property name="text" >
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0" >
    <widget class="QLabel" name="m_pipelineDir_2" >
     <property name="text" >
      <string>Pipeline Directory:</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1" >
    <widget class="QLineEdit" name="m_pipelineDir" >
     <property name="toolTip" >
      <string>Directory where all pipeline files will be saved</string>
     </property>
    </widget>
   </item>
   <item row="7" column="2" >
    <widget class="QToolButton" name="m_setPipelineDir" >
     <property name="text" >
      <string>...</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
